LiskHQ/lisk-sdk
# Update bootstrap with new oclif version and configs - Closes #7601

### What was the problem?

This PR resolves #7601 

### How was it solved?

- Remove community identifier
- Remove minFeePerByte
- Update oclif to use latest version

### How was it tested?
1. insert below into `commander/src/bootstrapping/templates/lisk-template-ts/generators/init_generator.ts:L94`. It needs this since commander and sdk are not published
```
this.spawnCommandSync('yarn', ['link', 'lisk-sdk'], {
	cwd: this.destinationRoot(),
});
this.spawnCommandSync('yarn', ['link', 'lisk-commander'], {
	cwd: this.destinationRoot(),
});
```
2. in `sdk` and `commander` folders, `yarn link`
3. in commander,
```
./bin/run init ~/Desktop/temp
```
4. check `temp` project is buildable and all the files are present
5. `/path/to/commander/bin/run generate:module mod`
6. `/path/to/commander/bin/run generate:command mod com`
